src/driver.c

```diff
@@ -399,4 +404,23 @@ void Driver_Step(void)
     GPIO_WriteReverse(LED_GPIO_PORT, (GPIO_Pin_TypeDef)LED_GPIO_PIN);
 #endif
 }
+
+/**
+ * @brief  Fill Rx Buffer in ISR Context
+ *
+ * @details  Invoked from Rx ISR
+ */
+ 
+void Get_Rx_IT(void)
```

Comment:
I have been trying to transition to a naming convention that incorporates the module (file) name on extern functions along the lines of "Driver_get_foo_bar()".

Reply:
Thanks for the feedback. I've updated the function name accordingly.